sources/packages/web/src/router/AESTRoutes.ts

```diff
@@ -161,6 +162,15 @@ export const aestRoutes: Array<RouteRecordRaw> = [
               clientType: ClientIdType.AEST,
             },
           },
+          {
+            path: AppRoutes.CASSupplierInformation,
+            name: AESTRoutesConst.CAS_SUPPLIER_INFORMATION,
+            props: true,
```

Comment:
Can we enforce the proper parser for new routes added? Similar to the below for institution?
```ts
props: {
  default: (route) => ({
    locationId: parseInt(route.params.locationId as string),
  }),
},
```